requirements.txt

```diff
@@ -2,3 +2,5 @@ requests
 gunicorn
 yaep
 pycountry
+python-Levenshtein
```

Comment:
It's listed as an optional dependency in the module documentation. Looking at the code it seems to try and import Levenshtein based sequence matching functions and fall back to difflib if that fails:

https://github.com/seatgeek/fuzzywuzzy/blob/master/fuzzywuzzy/fuzz.py